### PR TITLE
Allow the user to specify electrode material names

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

File: cpp/source/default_inspector.cc [hunk 1]
```diff
@@ -44,7 +45,19 @@ extract_data_from_super_capacitor(EnergyStorageDevice *device)
...
         ptree->get<double>("geometry.cathode_electrode_thickness");
-    for (std::string const &electrode : {"anode", "cathode"})
+    // NOTE: by default let us assume that the electrode materials are called
+    // "anode" and "cathode" but allow for the user to change it.
+    // We might want to make a SuperCapacitor specific class derived from
+    // Geometry that would be aware of the material tags assigned to the
+    // electrode/separator/collector regions. In that case we would access the
+    // SuperCapacitor::_geometry here and pull this information.
+    // Nevertheless, this is needed for debug purposes and I am not going to
+    // bother add a test yet. As a reminder there will be this not plus a line
```
Comment (Rombur) [inline cpp/source/default_inspector.cc]: not -> note

**Comment by dalg24** [comment]
@Rombur You good with the change after I fix this?

**Comment by Rombur** [comment]
yes

**Comment by dalg24** [comment]
K I did amend and forced the push

**Comment by dalg24** [comment]
Will merge without waiting for Travis so the test will fail

**Pull request merged**